tools/identity-resolution/identity-resolution/Helpers/GitHubAadAliasConverter.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Net.Http;
+using Azure.Sdk.Tools.NotificationConfiguration.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Sdk.Tools.NotificationConfiguration.Helpers
+{
+    public class GitHubToAadAliasResolver
+    {
+        /// <summary>
+        /// Create a new GitHubToMsAliasResolver
+        /// </summary>
+        /// <param name="clientId">AAD Client/App Id</param>
+        /// <param name="clientSecret">AAD Client/App Secret</param>
+        /// <param name="tenantID">AAD Tenant Id</param>
+        /// <param name="logger">Logger</param>
+        public GitHubToAadAliasResolver(
+            string clientId,
+            string clientSecret,
+            string tenantID,
+            ILogger<GitHubNameResolver> logger)
+        {
+            try
+            {
+                this.opsAuthToken = new AadTokenCredential(clientId, clientSecret, tenantID).GenerateAadToken();
+            }
+            catch (Exception ex)
+            {
+                logger.LogError("Failed to generate aad token. " + ex.Message);
```

Comment:
I think we would want to bubble up the exception here. Or if we don't want the exception in the constructor, move all this logic into an `Initialize()` method.